package.json

```diff
@@ -8,22 +8,20 @@
   },
   "scripts": {
     "prepare": "husky install && yarn build",
-    "clean": "lerna run clean && lerna clean --yes && rm -rf node_modules",
     "lint": "eslint packages/ scripts/ --ext=js,mjs,ts",
     "format": "prettier --write '{packages,scripts}/**/*.{js,mjs,ts,json,md}'",
     "bundlesize": "bundlesize --config ./scripts/bundlesize/bundlesize.config.json",
-    "build": "lerna run build --ignore @lwc/perf-benchmarks --ignore @lwc/perf-benchmarks-components --ignore @lwc/integration-tests",
-    "build:performance": "yarn run build:performance:components && yarn run build:performance:benchmarks",
-    "build:performance:components": "lerna exec --scope @lwc/perf-benchmarks-components -- yarn build",
-    "build:performance:benchmarks": "lerna exec --scope @lwc/perf-benchmarks -- yarn build",
-    "dev": "lerna run dev --parallel --ignore @lwc/perf-benchmarks --ignore @lwc/perf-benchmarks-components --ignore @lwc/integration-tests",
-    "test": "jest --config ./scripts/jest/root.config.js",
+    "build": "nx run-many --target=build --all --exclude=@lwc/perf-benchmarks,@lwc/perf-benchmarks-components,@lwc/integration-tests",
+    "build:performance": "yarn build:performance:components && yarn build:performance:benchmarks",
+    "build:performance:components": "nx build @lwc/perf-benchmarks-components",
+    "build:performance:benchmarks": "nx build @lwc/perf-benchmarks",
+    "dev": "nx run-many --target=dev --all --parallel=999 --exclude=@lwc/perf-benchmarks,@lwc/perf-benchmarks-components,@lwc/integration-tests",
+    "test": "nx run-many --target=test --all --exclude=@lwc/perf-benchmarks,@lwc/perf-benchmarks-components,@lwc/integration-tests,@lwc/integration-karma",
```

Comment:
I'm not sure I understand this change. This will run `jest` in every individual package rather than just once. I think it's better to just run `jest` once, since it has its own parallelism under the hood, and we probably don't want NX to skip running the tests if the source files are unchanged or cache the test output. (Otherwise this defeats the purpose of `yarn test` – it might not run the tests.)

Reply:
After discussing with you and PM last week during plan, removed the jest tests as part of the nx targets.